homeassistant/components/zwave.py

```diff
@@ -17,6 +20,14 @@
     EVENT_HOMEASSISTANT_STOP, EVENT_PLATFORM_DISCOVERED)
 from homeassistant.util import convert, slugify
 
+# Try to set a sane default if libopenzwave is installed
+try:
+    DEFAULT_ZWAVE_CONFIG_PATH = os.path.join(os.path.dirname(
+        libopenzwave.__file__), 'config')
+except NameError:
+    DEFAULT_ZWAVE_CONFIG_PATH = os.path.join(sys.prefix, 'share',
```

Comment:
No need to default anything as without openzwave it's not going to work. 

Should we detect if openzwave doesn't exist in the setup and link to instructions?